rio/src/RIO/Prelude/Renames.hs

```diff
@@ -28,8 +32,12 @@ type GVector = GVector.Vector
 type LByteString = BL.ByteString
 type LText = TL.Text
 
-toStrictBytes :: LByteString -> ByteString
```

Comment:
I totally for we had this function!

Reply:
Nor did I, and I'm _using_ `RIO.ByteString.Lazy.toStrict`! Docs project pays off before it's even merged. :grin:

Reply:
Yes, forgot. I forgot the word I was typing.